addons/pandora/api.gd

```diff
@@ -81,13 +81,9 @@ func delete_category(category:PandoraCategory) -> void:
 func delete_entity(entity:PandoraEntity) -> void:
 	_entity_backend.delete_entity(entity)
 
-
-func get_entity(entity_id:String) -> PandoraEntity:
```

Comment:
oops!

Reply:
lol I was loosing my mind here trying to figure out what broke